frontend/src/libs/components/card/card.tsx

```diff
@@ -1,15 +1,24 @@
 import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { type IconName } from '#libs/types/types.js';
 
+// import { Icon } from '../components.js';
```

Comment:
Let's prevent commenting code in PR

Reply:
@v0ldemar01 it`s just for now. i will fix this when @Liza-Veis approve media query in the video